### PR TITLE
Chan

### PR DESCRIPTION
**Pull request by ChanCoronado**
- Hey bro, so eto na nga pinaltan ko lang naman yung SnakaAuto ng SnakeAuto - Chan

### CODE REVIEW

**Comment by brandon-adm** [comment]
wow

File: src/Game.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public class Game extends JPanel{
 	
 	Timer t;
 	
-	String direction = "down";
+	String direction = "right";
```
Comment (cjdeclaro) [inline src/Game.java:44]: Why is this changed?

File: src/Home.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 public class Home {
 	public static void main(String[] args) {
-		JFrame frm = new JFrame("Snake Auto");
+		JFrame frm = new JFrame("Snake Auto - Chan");
```
Comment (BRY4N-CODER) [inline src/Home.java:5]: Dong why this change

**Review by cjdeclaro — APPROVED** [review]

**Review by BRY4N-CODER — COMMENTED** [review]
This is not yet accepted :<

File: src/Home.java (hunk 2 above)
Comment (ChanCoronado) [reply src/Home.java:5]: dili ka man don oy ba
